assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_histogram/run.py

```diff
@@ -167,7 +168,9 @@ def _check_and_format_azure_endpoint_url(
         domain = domain[:-1]
 
     if not re.match(domain_pattern_re, domain):
-        raise RuntimeError(f"Invalid Azure endpoint domain URL: {domain}.")
+        err_msg = f"Invalid Azure endpoint domain URL: {domain}."
```

Comment:
This is for the code change, are you going to manage the component release yourself?

Reply:
no not sure how to release it - I think I wanted to add a couple more tests/bug fixes before that as well